### PR TITLE
ci: drop kubernetes versions below 1.18

### LINKED ISSUE #137: Deprecate kubernetes below 1.18
I think we should stop testing kubernetes versions below 1.18 to save ci build time.

Is anybody using renovate on such old version

### PR DESCRIPTION
**Pull request by viceice**
fixes #137

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -50,8 +50,6 @@ jobs:
       matrix:
         k8s:
           # from https://github.com/instrumenta/kubernetes-json-schema
-          - v1.14.10
-          - v1.15.7
           - v1.16.4
           - v1.17.4
```
Comment (JamieMagee) [inline .github/workflows/ci.yaml:54]: IMO we should drop these 2 as well.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -78,14 +77,12 @@ jobs:
...
-          - v1.14.10
-          - v1.15.12
-          - v1.16.15
-          - v1.17.11
-          - v1.18.8
-          - v1.19.3
-          - v1.20.2
-          #- v1.21.1 does not work yet
+          - v1.16.15 #renovate: kindest
+          - v1.17.11 #renovate: kindest
```
Comment (viceice) [inline .github/workflows/ci.yaml:81]: @JamieMagee Should we drop those too?

Comment (JamieMagee) [reply .github/workflows/ci.yaml:81]: Yep 😄

**Review by viceice — COMMENTED** [review]
ready

File: renovate.json [hunk 3]
```diff
@@ -28,6 +28,15 @@
       ],
       "depNameTemplate": "renovate/renovate",
       "datasourceTemplate": "docker"
+    },
+    {
+      "description": "Update kindest kubernets references",
```
Comment (JamieMagee) [inline renovate.json:33]:
```suggestion
      "description": "Update kindest kubernetes references",
```

**Pull request merged**